.github/workflows/ci.yml

```diff
@@ -175,13 +175,12 @@ jobs:
           | install -m 755 /dev/stdin /usr/local/bin/taplo
       - name: Run Taplo
         id: taplo
-        run: taplo fmt --check --diff --option indent_string="    "
```

Comment:
I agree with this change. I think sticking with defaults unless we have a reason otherwise is a sensible choice.